documentation/modules/configuring/proc-joining-node-pool-roles.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+//
+// assembly-config.adoc
+
+[id='proc-joining-node-pools-roles-{context}']
+= Transitioning to dual-role nodes
+
+[role="_abstract"]
+This procedure describes how to transition from separate node pools with broker-only and controller-only roles to using a node pool with dual-roles.
+If your Kafka cluster is using node pools with dedicated controller and broker nodes, you can transition to using a single node pool with both roles.
+To do this, add the `broker` role to the controller-only node pool, rebalance the cluster to move partition replicas to the dual-role node pool, and then delete the old broker-only node pool.
+
+In this procedure, we start with two node pools `pool-a`, which has only the `controller` role and `pool-b` which has only the `broker` role:
+
+.Dual-role node pool
```

Comment:
Should this be something like `Dedicated role node pool` or `Single role node pool` if it's describing specs below?